RocketApp/Infrastructure/Tests/MotionClientTests/MotionClientTests.swift

```diff
@@ -0,0 +1,8 @@
+//
```

Comment:
Co tady dela ta hlavicka?

Reply:
Testy uz vlastne nebudou kdyz pouzivam managera od pointfree. Smazu to.